reana_server/utils.py

```diff
@@ -44,6 +45,10 @@ def create_user_workspace(user_workspace_path):
     reana_fs = fs.open_fs(app.config['SHARED_VOLUME_PATH'])
     if not reana_fs.exists(user_workspace_path):
         reana_fs.makedirs(user_workspace_path)
+    if os.environ.get("VC3USERID", None):
```

Comment:
You can use [`os.getenv()`](https://docs.python.org/3/library/os.html#os.getenv) directly and there's no need to specify `None` as it's the default :)